### DISCUSSION TITLE
Vavrycuk's conformal metric model

### DISCUSSION THREAD

**Discussion by ExpEarth** [a-cosmology-group/acg#108] [Cosmology Models]
There is an interesting paper by Vaclav Vavrycuk which was mentioned earlier in acg, but never discussed in great detail: _Cosmological Redshift and Cosmic Time Dilation in the FLRW Metric:_   [https://www.frontiersin.org/articles/10.3389/fphy.2022.826188/full](url)  In this paper he argues that the FLRW metric is the incorrect one to use in describing an expanding universe, because it only includes a term for space expansion, not for time dilation.  If you include the time dilation term, which is similar to the (1 + _z_) scale term, then it removes the need to have dark energy to explain supernova time dilation.  He also says that the FLRW metric does not even properly explain the Hubble redshift, but his "conformal" metric can.  I think his paper would also account for the new findings implying time dilation in remote quasars, as discussed in another thread.  He mentions that time dilation has already been seen in GRBs.  Vavrycuk basically says that the Hubble redshift is actually a gravitational redshift.

The paper requires a fair bit of general relativity.  Are there members here who can more or less crunch it?  Also, he still says that objects were closer together back in earlier times, as in the BB model.  What would happen if one used **only** the time expansion term and omitted the space expansion term?  Might he get a static model then?

**Comment by HanDeBruijn** [comment]
Clickable: [Cosmological Redshift and Cosmic Time Dilation in the FLRW Metric](https://www.frontiersin.org/articles/10.3389/fphy.2022.826188/full)

**Reply by budrap00** [reply]
```
Both metrics are based on the assumption of perfect isotropy and homogeneity and they satisfy the GR equations...

The physically meaningful coordinates should be identified with the “cosmological coordinate system,” in which all fundamental bodies are in rest
```
So the same old erroneous, 100 year old assumptions that remain unsupported by any empirical evidence are supposedly resuscitated by a new and improved universal metric. The point, of course, being to somehow maintain a classical (non-relativistic) Universe for metaphysical comfort and mathematical convenience. 

Tinkering with the math of a conceptually broken qualitative model is just pouring the same bitter wine into a shiny new bottle in hopes of improving the flavor. The new conformal metric is physically meaningless math that has nothing to do with the Cosmos we observe. Until all three empirically baseless cosmological assumptions (unity, homogeneity, isotropy) are discarded modern cosmology will remain a qualitatively absurd, unscientific mathematical freak show.

**Reply by ExpEarth** [reply]
Bud, it's not enough just to dump all over the various metrics that have been proposed.  I'm finding that unless one states a clear metric in GR for a cosmological model that reviewers at journals don't even think you have stated a model!  It's like there is no communication.  So we have no choice but to wade through these things.

**Reply by budrap00** [reply]
Matt, if your only aim is to get published by journals that will not allow you to question the qualitatively erroneous, foundational assumptions of the standard model, then what hope have you of overturning that model? If you come up with a different universal metric it will still be incompatible with GR which was derived to function in the absence of a universal frame, and your model will no more resemble the observed Cosmos than the current standard model. It will be as fundamentally ill-founded as LCDM, just wrong in a different way.

**Reply by ExpEarth** [reply]
Bud, I think that even if we question the main elements of the standard model (Big Bang, expansion, etc) and come up with a different model, that that new model will still need to have a way of describing it in GR.  If it's missing that description, then almost all cosmologists and physicists will not be able to follow the different model presented.  For your own model, by example, does it have a GR description?  If not I think it will be completely ignored.

**Reply by budrap00** [reply]
Matt, The problem is with what it is you are trying to describe. If you are trying to describe the same or similar, unified, coherent, simultaneous entity that is the default Universe of FLRW, you are describing an imaginary entity which cannot possibly exist given the limitation of the maximum light speed and the sheer scale of the observed Cosmos. You can describe such an entity with math or with metaphysics but it has no existence in empirical reality where its presence is precluded by the fundamental nature of the physical reality we actually observe. 

The Cosmos is not a Universe. The assumption that it is was not unreasonable 100 years ago when the known scale of the Cosmos was essentially our galaxy but it makes no physical sense now that our observations extend out 10 billion plus years in all directions with no boundary yet in evidence. GR was derived in the context of the solar system where it works well, but it does not work well at galactic scales and beyond. Why does anyone think it should be applicable to a proper accounting of cosmological behavior?

**Reply by HanDeBruijn** [reply]
@ExpEarth . Matt, you can publish **here** at this forum. And you are not being ignored by us. Isn't that good enough?
Publishing in a refereed journal makes me think of *[Raiders of the Lost Ark](https://en.wikipedia.org/wiki/Raiders_of_the_Lost_Ark)*, the ending:
[ .. ] *for "top men" to study. In a vast warehouse, the Ark* [your article] *is crated up and stored among countless other crates.*

**Reply by ExpEarth** [reply]
Han, as someone who works in an academic library, I would say no: you really need to store those articles for future readers!  The bigger problem nowadays is that the print archives are rapidly diminishing and the electronic ones can disappear in an instant.  As for our group here, we are all working on different models to escape the Big Bang.  Our thoughts on this topic have a great diversity and it is thus harder for us to reach any kind of consensus on anything.  The odd truth is that we are more likely find support for our models outside of this group than within!

**Reply by RedshiftDrift** [reply]
> The odd truth is that we are more likely find support for our models outside of this group than within!

There's nothing odd about this - this group's purpose is to share observational data that highlights problems with the Big Bang model.

**Comment by HanDeBruijn** [comment]
> He mentions that time dilation has already been seen in GRBs.

@marmetl . I thought that we have come to another conclusion in ["Detection of the cosmological time dilation of high-redshift quasars"](https://github.com/orgs/a-cosmology-group/discussions/96).

**Reply by ExpEarth** [reply]
From a section of the paper:

> For example, Zhang [61] studied a sample of 139 SWIFT long gamma-ray bursts (GRBs) with redshift z ≤ 8.2 and obtained a significant correlation between their duration and redshift. Similarly, Littlejohns and Butler [62] analysed 232 GRBs detected by the Swift/Burst Alert Telescope (BAT) and revealed that the observed durations are consistent with cosmic time dilation.

**Comment by HanDeBruijn** [comment]
From the reference:
> It is proved that the change in the frequency of redshifted photons is always connected with time dilation.

Yes of course. That's trivial. How are we measuring time? With an atomic clock. What are atomic clocks made from? They are made of photons. As Google says:

> Nowadays, Cesium-133 is used as the definition for the second due to the reliable frequency of microwave it emits. The definition is: The second is the duration of 9,192,631,770 periods of the **radiation** corresponding to the transition between the two hyperfine levels of the ground state of the caesium 133 atom.

Now what happens if all photons emitted become redshifted? Then it is observed that ~~the accompanying~~ atomic clocks run slower. Which is nothing else than the _definition_ of time dilation. Right?

**Reply by budrap00** [reply]
Han, what do you mean by "accompanying atomic clocks"?

**Comment by HanDeBruijn** [comment]
With help of the Variable Mass Hypothesis (**VMH**) in Arp and Narlikar's [FLAT SPACETIME COSMOLOGY](http://www.alternatievewiskunde.nl/cosmology/narlikar.pdf) it is possible to **get rid of [General Relativity](https://github.com/orgs/a-cosmology-group/discussions/78)**, at least for our purpose. Quote (p. 52): GR _solutions may be of relevance to **local** regions containing compact massive objects. [ .. ] Since this paper deals with **cosmological** effects we will confine our attention to the **[simple model](https://github.com/orgs/a-cosmology-group/discussions/63#discussioncomment-6255770)** [ .. ] Finally, the **Euclidean flat spacetime** becomes a natural, primary reference frame in which cosmological processes are most simply described._ (p. 56)

I think that Vavrycuk's model is somehow equivalent with Arp and Narlikar's theory. Let's take a look at just one formula in the paper, in comparison with our favorite [simple model](https://github.com/orgs/a-cosmology-group/discussions/63#discussioncomment-6255770).

$$
1+z=\frac{a(r)}{a(e)} \quad (1) \quad \Longleftrightarrow \quad \frac{a_0}{a(t)} = e^{H(t_0-t)} = \frac{m_0}{m} = 1+z \quad (\mbox{VMH})
$$

No confusion anymore with _conformal metric_ or _comoving and proper coordinates_. And what to think about the following sentences.
>The physically meaningful coordinates should be identified with the "cosmological coordinate system," in which all fundamental bodies are in rest.
Hence, the key for understanding Eqs 4, 5 is to define, which quantities are physical (being related to the cosmological coordinate system) and which quantities describe just an arbitrary coordinate with no physical meaning.

So I don't feel the need to _more or less crunch_ the GR formulas in the paper. I'm not impressed anyway:
![003](https://github.com/a-cosmology-group/acg/assets/135433222/973f5347-7940-4573-94ea-442550c7b7e1)

**Reply by budrap00** [reply]
`Finally, the Euclidean flat spacetime becomes a natural, primary reference frame in which cosmological processes are most simply described. `

A Euclidean spacetime is indistinguishable from a purely relational spacetime - it has no observable characteristics. So how can something unobservable be used as a reference frame, except in theory of course.

**Reply by ExpEarth** [reply]
I basically agree with you, Han, that GR should only be really necessary in describing compact massive objects.  On the cosmic scale Newtonian gravity in Euclidean space should suffice.  But cosmologists are so wedded to the idea of the Big Bang compact object origin that they feel GR must always be used.  So even if one says there never was such a compact state, most of them will still look for the metrics, equations of state, etc.

On Arp's model, it's good that you and some others are still working on it.  I did study it in the past and once wrote a review of _Seeing Red_ for _Apeiron_ .  Thanks for the link back to their paper.
